### PR TITLE
Add AWS CLI instructions.

### PR DESCRIPTION
**Pull request by enozcan**
Adds AWS CLI instructions to #1

### CODE REVIEW

File: doc/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -128,7 +456,51 @@ image::ss5.5-create-instance.png[]
...
++
+--
+Using the id's and the names you set above, now run two EC2 instances. In addition to the steps so far, you need to have
+a https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[key pair] and pass it via `--key-name` flag.
+The command below will start two Amazon Linux machines with type t2.micro and assign public IP addresses to each.
+Also a tag named "cluster-tag" with "guide-ec2-cluster" value will be assigned to the instances:
 
+[source, shell]
+----
+$ aws ec2 run-instances --image-id ami-04d29b6f966df1537 \
```
Comment (leszko) [inline doc/modules/ROOT/pages/index.adoc]: Will this AMI ID always work? I tried and got this exception:
```
The image id '[ami-04d29b6f966df1537]' does not exist
```

Maybe the AMI IDs are different for different regions. I try it on `eu-central-1`.

File: doc/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -128,7 +456,51 @@ image::ss5.5-create-instance.png[]
...
+- i-xxxxxxxxxxxxxxxxx
+- i-yyyyyyyyyyyyyyyyy
+----
+
+Finally, fetch the public IPs of the instances to establish SSH connections in the next step:
+[source, shell]
+----
+$ aws ec2 describe-instances \
+    --instance-ids i-xxxxxxxxxxxxxxxxx i-yyyyyyyyyyyyyyyyy \
+    | grep PublicIpAddress
```
Comment (leszko) [inline doc/modules/ROOT/pages/index.adoc]: An alternative to grep is `--query 'Reservations[*].Instances[*].PublicIpAddress'`. Up to you.

**Review by leszko — COMMENTED** [review]
Good guide. Added two comments.

File: doc/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (enozcan) [reply doc/modules/ROOT/pages/index.adoc]: I just noticed that they are region specific. I'll use parameters like `resolve:ssm:/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2` instead.

File: doc/modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (enozcan) [reply doc/modules/ROOT/pages/index.adoc]: I was looking for that, will update accordingly. Thanks.

**Review by leszko — APPROVED** [review]

**Comment by enozcan** [comment]
Merging this into #1 and let @hasancelik review both there.

**Pull request merged**